site/conf.py

```diff
@@ -1416,9 +1416,18 @@
     },
 }
 
-ADVERTISING = {
+PROPAGANDA = {
     "source_folder": "../propaganda",
-    "image_width": 180,
-    "image_heigth": 120
-
-}
+    "task0": {
+        "image_width": 180,
+        "image_heigth": 120,
+        "image_name": "logo_c.png",
+        "generated_at": "propaganda.html",
```

Comment:
use "propaganda_c" here (as you used "propaganda_r" below) so later is clear and explicit, when this html is used, which one it is